src/frontend/src/components/NavBar/components/AppNavBar/components/PathogenTabs/index.tsx

```diff
@@ -11,15 +16,18 @@ type PathogenTabEventHandler = (
 
 export const PathogenTabs = (): JSX.Element => {
   const router = useRouter();
-
   const currentPathogen = useSelector(selectCurrentPathogen);
 
   const handleTabClick: PathogenTabEventHandler = (_, value) => {
     const currentRoute = router.asPath;
-    const newRoute = currentRoute.replace(
-      /pathogen\/.*?(\/|$)/,
-      `pathogen/${value}/`
-    );
+
+    const groupRegex = new RegExp(`${GROUP_URL_INDICATOR}\\/.*?(\\/|$)`);
+    const groupUrlMatch = currentRoute.match(groupRegex);
+    const groupUrl = groupUrlMatch ? groupUrlMatch[0] : "";
+
+    // When the user changes between pathogens, we always start them on the samples page
+    const newRoute = `${ROUTES.DATA_SAMPLES}/${groupUrl}${PATHOGEN_URL_INDICATOR}/${value}`;
```

Comment:
I don't think this should be necessary. I think if you set the current pathogen in redux, the path should automatically update to reflect that.

```suggestion
  const currentPathogen = useSelector(selectCurrentPathogen);
  const dispatch = useDispatch();

  const handleTabClick: PathogenTabEventHandler = (_, value) => {
    dispatch(setPathogen(value));
  };
```